### PR TITLE
LRDOCS-8259 Web Content Display organization and links

### PR DESCRIPTION
**Pull request by jorge-campo**
This PR proposes the following changes, as described in [LRDOCS-8259](https://issues.liferay.com/browse/LRDOCS-8259):

- Review and update of Using the Web Content Display Widget article.
- A new organization of the cards for the Displaying Content landing page.

In the reorganization of cards, I grouped under a new __Additional Content Display Options__ section the display options with a single article. This new layout simplifies the landing page and provides a better organization of the standalone display options.

The markdown files, folders, cross-links, icons, and RST files were updated to reflect these changes.

### CODE REVIEW

**Comment by jorge-campo** [comment]
See _before_ and _after_ for the layout changes in LRDOCS-8259:

Capture 2021-02-09 at 12 29 47
Capture 2021-02-09 at 3 37 26

File: docs/dxp/7.x/en/content-authoring-and-management/web-content/web-content-articles/adding-a-basic-web-content-article.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Adding a Basic Web Content Article
 
+<<<<<<< HEAD
 Web content is one of the most practical and essential methods for creating site content. Web content articles can be displayed through [Fragments](../../../site-building/displaying-content/using_fragments.html), [Display Pages](../../../site-building/displaying-content/using-display-page-templates/creating-and-managing-display-page-templates.md), or [widgets](../../../site-building/displaying-content/using-the-web-content-display-widget.md).
+=======
+Web content is one of the most practical and essential methods for creating site content. Web content articles can be displayed through [widgets](../../../site-building/displaying-content/additional-content-display-options/using-the-web-content-display-widget.md), [Fragments](../../../site-building/displaying-content/using_fragments.html), or [display pages](../../../site-building/displaying-content/using-display-page-templates/creating-a-display-page-template.md).
+>>>>>>> ed9402257 (LRDOCS-8259 Update cross-links for the files that were moved. - Fix merge conflict)
```
Comment (jrhoun) [inline docs/dxp/7.x/en/content-authoring-and-management/web-content/web-content-articles/adding-a-basic-web-content-article.md:7]: @jorge-campo FYI - looks like you missed resolving a conflict here.

Comment (jrhoun) [reply docs/dxp/7.x/en/content-authoring-and-management/web-content/web-content-articles/adding-a-basic-web-content-article.md:7]: I went ahead and resolved the conflict locally already so no need to push up an update.

**Comment by jrhoun** [comment]
Merged. @jorge-campo see my changes 9e9464a90d1c71b646e3c7c4e15c23b409ac0c56 and e86cc05c8fdf71b5bf7a1c04b541aacc4271e47d.

@sez11a you can close this one.

File: docs/dxp/7.x/en/content-authoring-and-management/web-content/web-content-articles/adding-a-basic-web-content-article.md (hunk 1 above)
Comment (jorge-campo) [reply docs/dxp/7.x/en/content-authoring-and-management/web-content/web-content-articles/adding-a-basic-web-content-article.md:7]: Thanks for fixing this @jrhoun!

**Pull request closed without merging**